docs/tutorials/estimator_vqe.ipynb

```diff
@@ -0,0 +1,289 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6aa68656",
+   "metadata": {},
+   "source": [
+    "# Variational Quantum Eigensolver with Estimator Primitive"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "15a4588c",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "The Variational Quantum Eigensolver (VQE) is an optimization routine for finding the ground state energy (i.e. lowest eigenvalue) of a Hamiltonain and is a considered to be a viable candidate for NISQ hardware. In this tutorial, we will go over how to use IBM Quantum Runtime to submit your VQE's to the cloud. Specifically, we will be looking at calculating the ground state energy of the $H_2$ molecule using the estimator primitive. "
```

Comment:
```suggestion
    "The Variational Quantum Eigensolver (VQE) is an optimization routine for finding the ground state energy (i.e. lowest eigenvalue) of a Hamiltonian and is a considered to be a viable candidate for NISQ hardware. In this tutorial, we will go over how to use IBM Quantum Runtime to submit your VQE's to the cloud. Specifically, we will be looking at calculating the ground state energy of the $H_2$ molecule using the estimator primitive. "
```

Reply:
`In this tutorial, we will go over how to use IBM Quantum Runtime to submit your VQE's to the cloud.`
We should consider rewriting this line, what are you trying to say here? Use the `qiskit-ibm-runtime` client library to submit VQEs to the cloud?

Reply:
> `In this tutorial, we will go over how to use IBM Quantum Runtime to submit your VQE's to the cloud.`
> We should consider rewriting this line, what are you trying to say here? Use the `qiskit-ibm-runtime` client library to submit VQEs to the cloud?

I changed this, but I am open to new ideas